mp/src/game/server/momentum/mom_system_saveloc.cpp

```diff
@@ -281,8 +319,24 @@ void CSaveLocSystem::LevelShutdownPreEntity()
             kvCPs->AddSubKey(kvCP);
         }
 
+        // Add startmarks
+        KeyValues *kvStartMarks = new KeyValues("startmarks");
+        for (int track = 0; track < MAX_TRACKS; track++)
+        {
+            char szTrackNum[10];
+            Q_snprintf(szTrackNum, sizeof(szTrackNum), "%09i", track);
```

Comment:
You don't need 9 decimals (e.g. `000000001`) for something going up to ~64. Just `%03i` here should work fine.

Reply:
This is very true, szTrackNum is now 4 chars long and %03i is used in place of %09i.